pkg/scheduler/scheduler_test.go

```diff
@@ -187,6 +188,9 @@ func TestSchedule(t *testing.T) {
 		wantInadmissibleLeft map[string]sets.Set[string]
 		// wantPreempted is the keys of the workloads that get preempted in the scheduling cycle.
 		wantPreempted sets.Set[string]
+
```

Comment:
Adding a comment would be helpful, `TODO: if PartialAdmission graduates to beta, we must remove this`.

Reply:
It makes sense. Thanks for clarifying!